### ISSUE TITLE
AUR is no longer visible

### ISSUE THREAD

**Issue by DN-debug** [vinifmor/bauh#179]
Hi @vinifmor,

**Description**
It don't know if it's a bug or a mistake from my end that I might have failed to see. I have tested the current version of Bauh [on both stable and staging]. 
The only problem is that AUR packages are no longer visible rest is working good as it should. However it was, in the previous versions. 

The Bauh package is build from PKGBUILD taken from AUR. The optional dependency list does mention the required packages to make AUR work which I have, as always.

Important to note:
1. **~/.cache/bauh/arch/aur/index.txt** not present after initialization
2. **~/.config/bauh/arch.yml** not present

**Software Environment**
bauh version: 0.9.16.RC-3 
O.S: Arch Linux / Peux OS [derivative]
Python version: 3.9.5
Installation method: pacman

![Screenshot from 2021-05-29 10-56-52](https://user-images.githubusercontent.com/75267055/120059069-aa3c1f80-c06c-11eb-9995-069b796bb519.png)

**Comment by vinifmor** [comment]
Hello @DN-debug , check if **pacman** and **wget** are properly installed. Also, does this file (`/etc/arch-release`) exist on your system ?

**Comment by DN-debug** [comment]
> Hello @DN-debug , check if **pacman** and **wget** are properly installed. Also, does this file (`/etc/arch-release`) exist on your system ?

Umm, I'm on vanilla arch right now, so pacman is obviously present. "Wget" is present and m able to fetch files properly 

![Screenshot from 2021-05-31 18-57-25](https://user-images.githubusercontent.com/75267055/120200532-1459ed80-c242-11eb-91b2-28b9cfee04d9.png)

![Screenshot from 2021-05-31 19-16-04](https://user-images.githubusercontent.com/75267055/120202832-b1b62100-c244-11eb-89a2-70b4f4f1f94b.png)

, and yes arch-release is also present, I thought of it too and but ignored it because I should mention that on Peux OS [arch-release wasnt present], Bauh was able to detect aur which is why m sure I am missing something this time. 

It's not so important atm but I am going to test it on another machine maybe later today, which has vanilla arch. 

btw thanks!

**Comment by vinifmor** [comment]
That's it. Since the file does not exist o Peux OS, bauh will not enable Arch/AUR package management. This file is a standard for Arch-based distros. So, in your case, just create this empty file `/etc/arch-release` and it should work as expected. Also, AUR package management requires **git** installed (forgot to mention on the previous post).

**Comment by DN-debug** [comment]
> That's it. Since the file does not exist o Peux OS, bauh will not enable Arch/AUR package management. This file is a standard for Arch-based distros. So, in your case, just create this empty file `/etc/arch-release` and it should work as expected. Also, AUR package management requires **git** installed (forgot to mention on the previous post).

I think you misunderstood what I said, I am on vanilla arch not on Peux atm. and arch-release is present. It is working on one of the Peux flavors without arch-release is what I mentioned above. :-)

So this is the situation:
1. Peux KDE has bauh and recognized aur packages [doesnt have arch-release]
2. I recently released twm editions- bauh is the main software center in those editions and aur is not detected.
3. vanilla arch, my current system - bauh is again the main software center, no aur detected , arch-release present.

git is something i cant live without, so yes it is present too ;)
But i get what u said, and I know what to do, brb in 5 min with test results.

![Screenshot from 2021-05-31 20-21-50](https://user-images.githubusercontent.com/75267055/120210833-dbc01100-c24d-11eb-8123-cf249e2ac061.png)

![Screenshot from 2021-05-31 20-26-52](https://user-images.githubusercontent.com/75267055/120211449-92bc8c80-c24e-11eb-9fc5-2e2e1f8d7773.png)

**Comment by DN-debug** [comment]
ok m back, so here's what I did:


Case 1 : on Peux KDE machine

a. went back to check why it was working, well stupid me, i had linked arch-release to peux-release which is why it was working.

Case 2 : on Vanilla Arch- machine

a.  removed all of the release files
b. updated the system again and got the file back.
c. reboot
d. after login launched bauh and aur is working.

Case 2: on Peux TWM machine

a. same as above
b. ln -sf /etc/peux-release /etc/arch-release , and now I dont have to worry about neofetch overriding my terminal info. 
c. launched bauh - aur working


All done, thanks to u, coz my mind isnt working for several days now ;) 

Closing aye.
